src/adapters/postgres/cohort-adapter.ts

```
@@ -821,17 +957,44 @@ export class PostgresCohortService {
+              switch(type) {
+                case 'text':
+                case 'textarea':
+                case 'file':
+                case 'radio':
+                  return `(f."name" = '${key}' AND (fv."${type}Value" ILIKE '%${value}%' OR fv."value" ILIKE '%${value}%'))`;
+                case 'number':
+                  return `(f."name" = '${key}' AND (fv."numberValue" = ${value} OR fv."value" = '${value}'))`;
+                case 'calendar':
+                  return `(f."name" = '${key}' AND (fv."calendarValue"::text LIKE '%${value}%' OR fv."value" LIKE '%${value}%'))`;
+                case 'checkbox':
+                  const boolValue = value === 'true' || value === '1' || value === true;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)